### DISCUSSION TITLE
[Orc8r] Warning network orc8r_default is not found during build.py

### DISCUSSION THREAD

**Discussion by MGAbdElKhalek** [magma/magma#5423] [Q&A] [answered]
When I build the orc8r form the master as described in the quick start guide I see this warning at the end of the log
```
./build -a
...
...
removing Network orc8r_default
warning: Network orc8r_default is not found
```

**Comment by hcgatewood** [comment, marked as answer]
tl;dr this should be safe to ignore.

Context: [Networking in docker-compose](https://docs.docker.com/compose/networking/).

Docker-compose handles creating a custom network for the project. In this case, the created network is named `orc8r_default`. You can view all networks via `docker network ls`. When `docker-compose down` is run, it removes the network it expects to exist. If that network doesn't exist, it logs the warning you describe. When you run `./run.py`, which forwards to `docker-compose up -d`, the network gets re-created. So, from this perspective, everything is fine.

The one point I'm unclear on is why you're getting this warning from running `build.py -a` -- because that shouldn't run `docker-compose down`. Can you provide the following, so I can verify everything's working as expected @MGAbdElKhalek 

- Output from `git rev-parse HEAD ; git rev-parse --abbrev-ref HEAD`
- Full output from the `build.py -a` command

**Reply by MGAbdElKhalek** [reply]
Thanks @hcgatewood for your detailed reply which I totally agree with.
Please find the requested info. below
```
~/magma/orc8r/cloud/docker$ git rev-parse HEAD
fd312845f4a14ac375caf0d3793d57373c225946
```
```
~/magma/orc8r/cloud/docker$ git rev-parse --abbrev-ref HEAD
master
```
The output of `./build -a` > [orc8r_build_log.txt](https://github.com/magma/magma/files/6132268/orc8r_build_log.txt)

**Reply by hcgatewood** [reply]
Perfect. Yes if you want to open a PR to avoid this warning in the future, we want to remove this line https://github.com/magma/magma/blob/4d2c4df883d63fd87ce0d8ab7d6a2dae8576aade/orc8r/cloud/docker/build.py#L107
